task_DEV_5/task_DEV_5/LineSorter.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.IO;
+using System.Text;
+namespace task_DEV_5
+{
+    class LineSorter
+    {
+        private string path;
+        private FileStream stream;
+
+        public LineSorter(string path)
+        {
+            this.path = path;
+        }
+
+        private string ReadString()
+        {
+            StringBuilder builder = new StringBuilder();
+            char symbol;
+            while ((symbol = (char)stream.ReadByte()) != '\n' || stream.Length == stream.Position)
+            {
+                builder.Append(symbol);
+            }
+            return builder.ToString();
+        }
+        private void Recording(string firstString, string secondString)
+        {
+            byte[] bytes = Encoding.Default.GetBytes(firstString + '\n');
+            byte[] toBytes = Encoding.Default.GetBytes(secondString + '\n');
+            stream.Seek(-(bytes.Length + toBytes.Length), SeekOrigin.Current);
+            stream.Write(bytes, 0, bytes.Length);
+            stream.Write(toBytes, 0, toBytes.Length);
+        }
+
+        private void SortBubble(string firstString, string secondString)
```

Comment:
Remind please to discuss this moment in the our call.

Reply:
Done.